openmetadata-docs/content/v1.7.x-SNAPSHOT/getting-started/day-1/hybrid-saas/hybrid-ingestion-agent.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Hybrid Ingestion Agent
+slug: /getting-started/day-1/hybrid-saas/hybrid-ingestion-agent.md
+collate: true
+---
+
+# Hybrid Ingestion Agent
+
+The **Hybrid Ingestion Agent** is a component designed to enable Collate customers operating in hybrid environments to securely execute ingestion workflows within their own cloud infrastructure. In this setup, your SaaS instance is hosted on Collate’s cloud, while tools like Argo Workflows are deployed within the your private cloud. The Hybrid Agent acts as a bridge between these two environments, allowing ingestion workflows to be triggered and managed remotely—without requiring the customer to share secrets or sensitive credentials with Collate. It securely receives workflow execution requests and orchestrates them locally, maintaining full control and data privacy within the customer’s environment.
+
+## Prerequisites
+
+Before setting up the Hybrid Ingestion Agent, ensure the following:
+
+- Your SaaS instance is up and running.
+- You have **Argo** deployed in your own cloud environment.
+- Network access between the customer’s cloud and Collate’s WebSocket server (outbound traffic from your cloud to Collate).
+- Java 17+ for the agent.
+- Docker.
+- Secrets manager configured on the Customer cloud.
+
+## How to Activate the Hybrid Ingestion Agent Server
+
+The server-side WebSocket component runs as part of the Collate server within Collate's cloud and handles the orchestration of ingestion triggers. To activate the hybrid ingestion agent server, some environment variables must be configured:
+
+
+| Environment Variable                                | Value                                        |
+|-----------------------------------------------------|----------------------------------------------|
+| `OM_SM_ACCESS_KEY_ID`                               | Secrets manager key ID                       |
+| `OM_SM_ACCESS_KEY`                                  | Secrets manager access key                   |
+| `OM_SM_REGION`                                      | Secrets manager region                       |
+| `PIPELINE_SERVICE_CLIENT`                           | `argo`                                        |
+| `PIPELINE_SERVICE_CLIENT_CLASS_NAME`                | `io.collate.service.hybrid.HybridServiceClient` |
+| `PIPELINE_SERVICE_CLIENT_SECRETS_MANAGER_LOADER`    | `env`                                         |
+
+
+Ensure that the Collate's server port is accessible and configured for secure communication if used in production.
+
+## How to Run the Hybrid Ingestion Agent
+
+On the customer’s cloud environment, run the Hybrid Agent which connects to the Collate's server and wraps calls to the Argo Workflows client.
```

Comment:
Not sure if the subject is changed on purpose here. If this is still addressing the user then it needs to be changed. Otherwise disregard.

```suggestion
On your cloud environment, run the Hybrid Agent which connects to the Collate's server and wraps calls to the Argo Workflows client.
```

Reply:
If this is meant to be read by the end user, the documentation should be addressing them (using second person "you/your")